xamarin/Xamarin.Forms
# Fix 7709 | Fix showing keyboard when changing placeholder in entry focused event

### Description of Change ###
I handle setting of placeholder for entry. As I see described keyboard hiding persists on native Android development. So to conduct development experience on both platforms I handle it on Android renderer.

### Issues Resolved ### 
- fixes #7709

### API Changes ###
None

### Platforms Affected ### 
- Android

### Behavioral/Visual Changes ###
A keyboard will be shown correctly with Focused handler.

### Before/After Screenshots ### 
Before
![before_changes](https://user-images.githubusercontent.com/6042648/66715047-5c0ad780-edc7-11e9-907c-aa3f6c8b739c.gif)

After
![afterChanges](https://user-images.githubusercontent.com/6042648/66722704-d06e6680-ee19-11e9-8b9c-407746eedaba.gif)

### Testing Procedure ###
1. Add entry;
2. Add a handler for Focused event;
3. Set placeholder in event handler method;
4. Click on entry. You will notice that the text field has focus but the keyboard does not appear on the screen.

Entry x:Name="entTest" Placeholder="blabla" Focused="focus_event"

public void focus_event(object sender, FocusRequestArgs arg)
{
entTest.Placeholder = null;
}

### PR Checklist ###
